### ISSUE TITLE
Add `parameterList` scope type

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#493]
When inside a function, saying eg `"take argue list"` will select the list of function parameters.  

- [ ] Make sure that when combined with #242, it will insert a comma after the last argument, or just move cursor to empty arg list if there are no args

**Comment by pokey** [comment]
Yes still good first issue. I removed "help wanted" label from all issues marked "good first issue" because to me the latter implies the former. Would be a bit misleading to label something "good first issue" if we don't want help with it 😄

**Comment by Will-Sommers** [comment]
Heyo @pokey — looking at this one again since I'm near it. Do we want this to be distinct from the `all` modifier? 

Also, some specifics. We'd like a single selection here? 
I'm curious about the check mark related to #242, ping me on slack?
